README.md

```diff
@@ -26,10 +26,9 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.2</b> or higher
+ * NodeJS 8.0 or higher
  * [Composer](https://getcomposer.org/)
```

Comment:
Should we remove composer here?  @kongtiaowang @driusan @johnsaigle  
since it is installed by `make`

Reply:
It's run by `make`, but not installed by `make`. It still needs to be installed prior to LORIS, similar to Apache, PHP, etc.